common/persistence/sql/sqlplugin/tests/queue_v2.go

```diff
@@ -659,3 +663,73 @@ func testSelectNameFromQueueV2NegativeToken(ctx context.Context, t *testing.T, b
 	require.Error(t, err)
 	require.ErrorIs(t, err, persistence.ErrNegativeListQueuesOffset)
 }
+
+func testListQueuesGetPartitionFails(ctx context.Context, t *testing.T, baseDB sqlplugin.DB) {
+	queueType := persistence.QueueTypeHistoryNormal
+	queueName := "test-queue-" + t.Name()
+	logger := &logRecorder{Logger: log.NewTestLogger()}
+	q := persistencesql.NewQueueV2(baseDB, logger)
+	queuePB := persistencespb.Queue{
+		Partitions: map[int32]*persistencespb.QueuePartition{
+			0: {},
+			1: {},
+		},
```

Comment:
https://github.com/temporalio/temporal/blob/ed20abbec977e71a033aeb38de7666de0c21639d/common/persistence/queue_v2_util.go#L131-L131
It is failing because our queue metadata cannot have more than one partition. That is the only way I can fail this function call: https://github.com/temporalio/temporal/blob/ed20abbec977e71a033aeb38de7666de0c21639d/common/persistence/sql/queue_v2.go#L486-L486

Reply:
Ok cool. Could you just add a short comment about it here in code?